pootle/apps/pootle_statistics/models.py

```diff
@@ -560,6 +561,8 @@ def get_score_delta(self):
         def get_sugg_rejected():
             result = 0
             try:
+                # Get similarity from initital submission where
```

Comment:
Typo: initial. See the rest of similar comments. Also you might add the end period.